lib/manager/pip_setup/extract.js

```diff
@@ -4,11 +4,41 @@ const { join } = require('upath');
 const { isSkipComment } = require('../../util/ignore');
 const { dependencyPattern } = require('../pip_requirements/extract');
 
+const pythonVersions = ['python', 'python3', 'python3.7'];
+let pythonAlias = null;
 module.exports = {
   extractPackageFile,
   extractSetupFile,
+  parsePythonVersion,
+  getPythonAlias,
+  pythonVersions,
 };
 
+function parsePythonVersion(str) {
+  const arr = str.split(' ')[1].split('.');
+  return [parseInt(arr[0], 10), parseInt(arr[1], 10)];
+}
+
+async function getPythonAlias() {
+  if (pythonAlias) {
+    return pythonAlias;
+  }
+  pythonAlias = pythonVersions[0]; // fallback to 'python'
+  for (const pythonVersion of pythonVersions) {
+    try {
+      const { stdout, stderr } = await exec(`${pythonVersion} --version`);
+      const version = parsePythonVersion(stdout || stderr);
+      // istanbul ignore if
+      if (version[0] >= 3 && version[1] >= 7) {
+        pythonAlias = pythonVersion;
+      }
+    } catch (err) {
+      logger.info(`${pythonVersion} alias not found`);
```

Comment:
Logger.debug not logger.info. Assuming it’s quite common to have python 2 as default then there’s no need to pollute the info log or confuse users